### DISCUSSION TITLE
Configuring PHP directives without custom images

### DISCUSSION THREAD

**Discussion by pompushko** [docker-library/php#1618] [General]
Hello

Is there any way to increase upload max file size (upload_max_filesize) in PHP.INI without creating own image?
Why in in 2k24 by default only 2mb?
Is it possible to use environment variables?

Thank you

**Comment by tianon** [comment]
https://github.com/docker-library/wordpress/discussions/879#discussioncomment-8310203

**Comment by pompushko** [comment]
> [docker-library/wordpress#879 (comment)](https://github.com/docker-library/wordpress/discussions/879#discussioncomment-8310203)

Thank you , but this way dont works in AWS ECS Fargate

**Comment by tianon** [comment]
You can't set a custom container command on Fargate? (without that, your only option is going to be to create your own image)

**Comment by pompushko** [comment]
> You can't set a custom container command on Fargate? (without that, your only option is going to be to create your own image)

Well... I can login into container, add some data in to php.ini like that:
```
echo "upload_max_filesize=256M" >> $PHP_INI_DIR/conf.d/upload.ini
echo "post_max_size=256M" >> $PHP_INI_DIR/conf.d/postsize.ini
echo "memory_limit=1024M" >> $PHP_INI_DIR/conf.d/memory.ini
echo "max_execution_time=120" >> $PHP_INI_DIR/conf.d/execution.ini
```

But if I will try to restart\reload apache2 - container restart automatically :D

**Comment by LaurentGoderre** [comment]
You can use volume mount to mount those config to the container
